app/pods/publications/index/template.hbs

```diff
@@ -170,6 +170,18 @@
                       </span>
                     </td>
                   {{/if}}
+                  {{#if this.tableColumnDisplayOptions.proofPrintCorrector}}
+                    <td class="auk-u-text-nowrap">
+                      {{#if row.publicationSubcase.proofPrintCorrector}}
+                        <Auk::Avatar @icon="user" @size="small">
+                          <p>{{row.publicationSubcase.proofPrintCorrectorInitials}} </p>
```

Comment:
De story description is hier misschien wat verwarrend geschreven. Je hoeft niets speciaal te doen voor de initialen. Je mag in de overzichtstabel gewoon weergeven wat er in de property `proofPrintCorrector` zit (en normaal gaat de gebruiker daar initialen in invullen, maar dat doet er an sich niet toe).